src/components/exercise9/fetching.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import UserComponent from "./userComponent";
+
+export default function Fetch() {
+  const [user, setUser] = useState([]);
+  const [random, setRandom] = useState(0);
+
+  useEffect(
+    function () {
+      fetch("https://random-data-api.com/api/users/random_user?size=10")
+        .then((res) => res.json())
+        .then((data) => setUser(data));
+      console.log(user);
+    },
+    [random]
+  );
+
+  let data = user.map((item) => (
+    <UserComponent
+      key={item.id}
+      firstName={item.first_name}
+      lastName={item.last_name}
+      avatar={item.avatar}
+      employment={item.employment.title}
+      skill={item.employment.key_skill}
+      userName={item.username}
+      email={item.email}
+      tel={item.phone_number}
+      gender={item.gender}
```

Comment:
```suggestion


     <UserComponent  key={item.id}  itemInfo={item}/>
```
I suggest that instead of passing all props like this, It would be better if you pass it as shown below

- This will not only make your codes short and clear but it will also reduce the lines of code written and save you much time to work on other tasks